gp-templates/settings.php

```diff
@@ -29,7 +29,7 @@
 ?>
 <h2><?php _e( 'Your Settings', 'glotpress' ); ?></h2>
 <form action="" method="post">
-	<?php include_once __DIR__ . '/settings-edit.php'; ?>
+	<?php require_once __DIR__ . '/settings-edit.php'; ?>
```

Comment:
What was the reason for this change?

Reply:
This was changed by this rule:
`PEAR.Files.IncludingFile.UseRequireOnce`